### PR TITLE
fix-bundleVersion

### PR DESCRIPTION
**Pull request by tleclere-jahia**
Generate snapshot module

### CODE REVIEW

File: src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java [hunk 1]
```diff
@@ -2,108 +2,106 @@
...
-    private static final String QUERY = "SELECT * FROM [jnt:moduleManagementBundle] as N WHERE ISDESCENDANTNODE(N, '/module-management/') "
-            + "AND N.[j:groupId]='%s' AND N.[j:symbolicName]='%s' AND N.[j:version]='%s'";
+    private static final String QUERY = "SELECT * FROM [jnt:moduleManagementBundle] WHERE ISDESCENDANTNODE('/module-management/')"
+            + " AND [j:groupId]='%s' AND [j:symbolicName]='%s' AND [j:version]='%s'";
 
     @Override
     public Object execute() throws Exception {
-        final String tmpPath = SettingsBean.getInstance().getTmpContentDiskPath();
-        final String zipPath = tmpPath + "/modulesExport" + System.currentTimeMillis() + ".zip";
+        SettingsBean settingsBean = BundleUtils.getOsgiService(SettingsBean.class, null);
```
Comment (fbourasse) [inline src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java]: Should be final

File: src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java [hunk 2]
```diff
@@ -2,108 +2,106 @@
...
+import javax.jcr.RepositoryException;
+import javax.jcr.query.Query;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
```
Comment (fbourasse) [inline src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java:31]: Why change the order of the imports?

Comment (tleclere-jahia) [reply src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java:31]: Intellij formatter

File: src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java (hunk 1 above)
Comment (tleclere-jahia) [reply src/main/java/org/community/provisioninggenerator/command/GenerateCommand.java]: Right, I update it

**Review by fbourasse — APPROVED** [review]
Some changes

**Pull request merged**